schema/home/home_page_artist_module.js

```diff
@@ -57,27 +65,31 @@ export const HomePageArtistModuleType = new GraphQLObjectType({
       },
     },
     key: {
+      description: 'Module identifier.',
       type: GraphQLString,
     },
     results: {
       type: new GraphQLList(Artist.type),
       resolve: ({ key, display, params }, options, { rootValue: { accessToken, userID } }) => {
-        return Results[key].resolve(accessToken, userID);
+        return HomePageArtistModuleTypes[key].resolve(accessToken, userID);
       },
     },
   },
 });
 
 const HomePageArtistModule = {
   type: HomePageArtistModuleType,
-  description: 'Single artist module to show on the home screen',
+  description: 'Single artist module to show on the home screen.',
   args: {
     key: {
-      type: GraphQLString,
-      description: 'Module key',
+      description: 'Module identifier.',
+      type: new GraphQLEnumType({
+        name: 'HomePageArtistModuleTypes',
+        values: HomePageArtistModuleTypes,
```

Comment:
The GraphQLEnumType constructor [picks the values](https://github.com/graphql/graphql-js/blob/8df7894c8974bd19c600ad05df9d42ed3def3e65/src/type/definition.js#L743-L749) it needs from the given config object, so I opted to move the `description` to the `HomePageArtistModuleTypes` definition as well to keep everything in one place.